src/main/resources/db/migration/V1_3__seed_service_category.sql

```diff
@@ -0,0 +1,9 @@
+insert into service_category (id, name) values ('d21fc249-41ff-4987-8027-0c559f9bbd2e', 'Accommodation');
+insert into service_category (id, name) values ('ca374ac3-84eb-4b91-bea7-9005398f426f','Education, Training and Employment (ETE)');
```

Comment:
does created timestamp get set when these are not created by hibernate?

Reply:
No.  Good spot - I will add that in